docs/snippets_generation/swagger_ui_curlify.py

```diff
@@ -11,7 +11,7 @@ def generate_sample(op_spec, data_params_are_present, model_name, full_spec, bas
     headers = dict(base_headers)
     # add mandatory token
     headers["Authorization"] = 'Bearer ${ACCESS_TOKEN}'
-
+    headers.pop('User-Agent', None)
```

Comment:
I'd rather recreate new `headers` here (or in the Jinja template) instead of passing them from the `ftd.py`. The connection class already has Ansible-specific header (and might have more in the future), so I'd not reuse it in the docs anymore.

Reply:
fixed